addons/mrp/mrp.py

```diff
@@ -541,6 +550,10 @@ def _mrp_from_move(self, cr, uid, ids, context=None):
             res += self.pool.get("mrp.production").search(cr, uid, [('move_lines', 'in', move.id)], context=context)
         return res
 
+    def _get_mo_ids(self, cr, uid, product_template_ids, context=None):
+        # It will return Manufacturing Orders ids of product.
+        return self.pool["mrp.production"].search(cr, uid , [('product_id.product_tmpl_id', 'in', product_template_ids)], context=context)
```

Comment:
this is a trigger function. it is called from product.template object. so self is product.template, not mrp.production.